maxim/logger/portkey/portkey.py

```
@@ -0,0 +1,184 @@
+
+from portkey_ai import AsyncPortkey, Portkey
+
+from ...scribe import scribe
+from ..logger import Generation, Logger, Trace
+from ..openai.utils import OpenAIUtils
+
+
+class MaximPortkeyClient:
+    """Maxim instrumenter for Portkey client that directly handles chat.completion method."""
+    
+    def __init__(self, client: Union[Portkey, AsyncPortkey], logger: Logger):        
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)